bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/dto/clip2/ProductData.java

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) 2010-2023 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.hue.internal.dto.clip2;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.openhab.binding.hue.internal.dto.ApiVersion;
+import org.openhab.binding.hue.internal.dto.clip2.enums.Archetype;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * DTO for CLIP 2 product data.
+ *
+ * @author Andrew Fiddian-Green - Initial contribution
+ */
+@NonNullByDefault
+public class ProductData {
+    private @NonNullByDefault({}) @SerializedName("model_id") String modelId;
+    private @NonNullByDefault({}) @SerializedName("manufacturer_name") String manufacturerName;
+    private @NonNullByDefault({}) @SerializedName("product_name") String productName;
+    private @NonNullByDefault({}) @SerializedName("product_archetype") String productArchetype;
+    private @NonNullByDefault({}) Boolean certified;
+    private @NonNullByDefault({}) @SerializedName("software_version") String softwareVersion;
+    private @Nullable @SerializedName("hardware_platform_type") String hardwarePlatformType;
+
+    public String getModelId() {
+        return modelId;
+    }
+
+    public String getManufacturerName() {
+        return manufacturerName;
+    }
+
+    public String getProductName() {
+        return productName;
+    }
+
+    public Archetype getProductArchetype() {
+        return Archetype.of(productArchetype);
+    }
+
+    public Boolean getCertified() {
+        return certified != null ? certified : false;
+    }
+
+    public ApiVersion getSoftwareVersion() {
+        return ApiVersion.of(softwareVersion);
```

Comment:
This needs to be more lenient. I get the following with a 3rd party device ([this one](https://de.paulmann.com/p/varifit-led-einbaupanel-veluna-ip44-eckig-215x215mm-17w-1300lm-white-switch-transparent/95380)):

```
15:30:30.111 [ERROR] [.core.thing.internal.ThingManagerImpl] - Exception occurred during bridge handler ('hue:clip2:79615b28b8') notification about handler initialization of child 'hue:device:79615b28b8:e1a29fc1-b58d-4635-8cce-956ccd8f60fe': Version "1400" is not valid
java.lang.IllegalArgumentException: Version "1400" is not valid
        at org.openhab.binding.hue.internal.dto.ApiVersion.of(ApiVersion.java:49) ~[?:?]
        at org.openhab.binding.hue.internal.dto.clip2.ProductData.getSoftwareVersion(ProductData.java:58) ~[?:?]
        at org.openhab.binding.hue.internal.handler.Clip2ThingHandler.updateProperties(Clip2ThingHandler.java:599) ~[?:?]
        at org.openhab.binding.hue.internal.handler.Clip2ThingHandler.onResource(Clip2ThingHandler.java:328) ~[?:?]
        at org.openhab.binding.hue.internal.handler.Clip2BridgeHandler.lambda$7(Clip2BridgeHandler.java:455) ~[?:?]
        at java.lang.Iterable.forEach(Iterable.java:75) ~[?:?]
        at org.openhab.binding.hue.internal.handler.Clip2BridgeHandler.onResource(Clip2BridgeHandler.java:452) ~[?:?]
        at org.openhab.binding.hue.internal.handler.Clip2BridgeHandler.lambda$1(Clip2BridgeHandler.java:226) ~[?:?]
        at java.util.ArrayList.forEach(ArrayList.java:1511) ~[?:?]
        at org.openhab.binding.hue.internal.handler.Clip2BridgeHandler.childHandlerInitialized(Clip2BridgeHandler.java:226) ~[?:?]
        at org.openhab.core.thing.internal.ThingManagerImpl.lambda$12(ThingManagerImpl.java:840) ~[?:?]
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539) ~[?:?]
        at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304) ~[?:?]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
        at java.lang.Thread.run(Thread.java:833) ~[?:?]
```

Reply:
Looking through my other devices:
- Hue and Tradfri follow the expected format (X.Y.Z)
- Paulmann doesn't
- Chinese dimmers don't either

Easiest solution would probably be treating the value `X` as `X.0.0`.

Reply:
I will just return a string.